service/src/main/java/de/adorsys/psd2/sandbox/xs2a/testdata/TestDataFileReader.java

```diff
@@ -0,0 +1,76 @@
+package de.adorsys.psd2.sandbox.xs2a.testdata;
+
+import de.adorsys.psd2.sandbox.xs2a.testdata.domain.Amount;
+import de.adorsys.psd2.sandbox.xs2a.testdata.domain.Transaction;
+import java.math.BigDecimal;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.time.LocalDate;
+import java.util.Currency;
+import java.util.HashMap;
+import java.util.List;
+import java.util.UUID;
+import java.util.stream.Collectors;
+import lombok.NoArgsConstructor;
+import org.springframework.stereotype.Component;
+
+@Component
+@NoArgsConstructor
+final class TestDataFileReader {
+
+  private static final String FILE_NAME = "transactions_dump.txt";
+
+  static HashMap<String, Transaction> readTransactionsFromFile() {
+
+    List<String> txs;
+    try {
+      txs = Files.lines(Paths.get(ClassLoader.getSystemResource(FILE_NAME).toURI()))
```

Comment:
There is a Spring way to do this with `ResourceLoader`. Should look a bit cleaner then.